### ISSUE TITLE
After the background optimization getting invalid value

### ISSUE THREAD

**Issue by gitim** [ben-eb/cssnano#139]
Hi! After the optimization of the the `background` shorthand getting invalid value for this case:
```css
background: none no-repeat left bottom 60px / auto 40%;
```
I am getting:
```css
background: none no-repeat 0 100% 60px/auto 40%;
```
which is invalid:
<img width="381" alt="bg" src="https://cloud.githubusercontent.com/assets/3778452/13021473/7417cc84-d1ec-11e5-86c3-726bc19d006f.png">

**Comment by TrySound** [comment]
@gitim Is your source example valid? Try without cssnano

**Comment by gitim** [comment]
@TrySound yes:
<img width="384" alt="2016-02-13 1 05 42" src="https://cloud.githubusercontent.com/assets/3778452/13021774/fc69ecd8-d1ed-11e5-90d0-2fc986109b97.png">

**Comment by ben-eb** [comment]
Does it work with this?

```
background: none no-repeat 0 100% 60px / auto 40%;
```

**Comment by gitim** [comment]
@ben-eb nope

**Comment by ben-eb** [comment]
Looks like the three value syntax for position isn't being respected properly. For now you can disable with `{reducePositions: false}`.

**Comment by ben-eb** [comment]
@gitim Fixed in https://github.com/ben-eb/cssnano/commit/4cf795978aa05ef4ddc987d27ace768ef5ab9318 & released in 3.5.2.

**Comment by gitim** [comment]
Cool, thanks!
